panacea/query_client.go

```diff
@@ -52,7 +54,20 @@ func NewQueryClient(ctx context.Context, chainID, rpcAddr string, trustedBlockHe
 		return nil, err
 	}
 	pvs := []provider.Provider{pv}
-	store := dbs.New(dbm.NewMemDB(), chainID)
+
+	if _, err := os.Stat(leveldbPath); os.IsNotExist(err) {
+		err = os.MkdirAll(leveldbPath, 0755)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Does this work well?
My guess is that the sgx execution environment won't find this file path.

The executable path in our sgx environment is:
`/data`
However, the path referenced here now seems to go down one level than `/data`.

You can refer to [this code](https://github.com/medibloc/panacea-doracle/blob/main/cmd/doracled/cmd/start.go#L17) to see how to read `config.toml`

Finally, it would be good if this data is saved in the following path in sgx environment.
```
$HOME/.doracle/data
```

Reply:
Thanks, I fixed it at https://github.com/medibloc/panacea-doracle/pull/12/commits/6fcd8d0559e59a7a72376281d104d78957840d43.
It would be better to change to put that `dbDir` in the config.
I will reflect it in the next PR.